content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -412,11 +412,19 @@ A PersistentVolume can be mounted on a host in any way supported by the resource
 
 The access modes are:
 
-* ReadWriteOnce -- the volume can be mounted as read-write by a single node
-* ReadOnlyMany -- the volume can be mounted read-only by many nodes
-* ReadWriteMany -- the volume can be mounted as read-write by many nodes
-* ReadWriteOncePod -- the volume can be mounted as read-write by a single Pod.
-  This is only supported for CSI volumes and Kubernetes version 1.22+.
+<dl>
+  <dt><code>ReadWriteOnce</code></dt>
+  <dd>: the volume can be mounted as read-write by a single node.ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node.</dd>
```

Comment:
Try this:
```suggestion
`ReadWriteOnce`
: the volume can be mounted as read-write by a single node. `ReadWriteOnce` access mode still can allow multiple pods to access the volume when the pods are running on the same node.
```
(and the same for other list entries)

Reply:
@sftim So  should i remove definition list tags <dl> <dt> from the text?